common/test/test_lint.py

```diff
@@ -77,6 +77,7 @@ def test_with_pylint(self):
             'E1101',  # no-member
             'W1401',  # anomalous-backslash-in-string (invalid escape sequence)
             'E0401',  # import-error
+            'I0021',  # useless-suppression
```

Comment:
See [pylint docu about `I0021`](https://pylint.readthedocs.io/en/latest/user_guide/messages/information/useless-suppression.html). It detects useless pylint-disable comments (e.g. `# pylint: disable=E0401`). I assume we will use such comments a lot in the next years. But they also become useless again very often because of heavy refactoring.